pkg/simpleschema/transform.go

```diff
@@ -212,8 +215,39 @@ func (tf *transformer) applyMarkers(schema *extv1.JSONSchemaProps, markers []*Ma
 			if val, err := strconv.ParseFloat(marker.Value, 64); err == nil {
 				schema.Maximum = &val
 			}
+		case MarkerTypeEnum:
+			enumValues := strings.Split(marker.Value, ",")
+
+			var enumJSONValues []extv1.JSON
+
+			for _, val := range enumValues {
+				val = strings.TrimSpace(val)
+
+				if val == "" {
+					return fmt.Errorf("empty enum values are not allowed")
+				}
+
+				var rawValue []byte
+
+				switch schema.Type {
+				case "string":
+					rawValue = []byte(fmt.Sprintf("\"%s\"", val))
```

Comment:
nit: you can use %q to quote-format a string 
```suggestion
					rawValue = []byte(fmt.Sprintf("%q", val))
```